ui/statistics/src/main/java/de/danoeh/antennapod/ui/statistics/StatisticsFragment.java

```diff
@@ -115,11 +110,8 @@ private void doResetStatistics() {
                 .putLong(PREF_FILTER_TO, Long.MAX_VALUE)
                 .apply();
 
-        Disposable disposable = Completable.fromFuture(DBWriter.resetStatistics())
-                .subscribeOn(Schedulers.io())
-                .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(() -> EventBus.getDefault().post(new StatisticsEvent()),
-                        error -> Log.e(TAG, Log.getStackTraceString(error)));
+        DBWriter.resetStatistics();
+        EventBus.getDefault().post(new StatisticsEvent());
```

Comment:
Why this? Now it does IO on the main thread

Reply:
Oops, I missed this somehow in the self-review. I was testing something and forgot to put it back.